Camera app can't open when "Do not disturb" (DND / Zen mode) is enabled.

Device: Xiaomi Mi 6X / wayne
Version: AOSP v102+

Steps:
* Enable Do not disturb mode.
* Open "Camera" app. (built-in, not 3rd-party)
* Boom!

Logs (from adb logcat):
```--------- beginning of main
08-30 12:07:42.251  3636  3636 W ServiceManagement: Waited one second for android.hardware.boot@1.0::IBootControl/default. Waiting another...
08-30 12:07:42.491  1864  2116 I vol.Events: writeEvent mute_changed STREAM_SYSTEM false
08-30 12:07:42.498  1864  2116 I vol.Events: writeEvent external_ringer_mode_changed normal
08-30 12:07:42.532  1864  2116 I vol.Events: writeEvent zen_mode_changed off
08-30 12:07:43.252  3636  3636 W ServiceManagement: Waited one second for android.hardware.boot@1.0::IBootControl/default. Waiting another...
08-30 12:07:43.402  4162  4203 W Notification: Use of stream types is deprecated for operations other than volume control
08-30 12:07:43.403  4162  4203 W Notification: See the documentation of setSound() for what to use instead with android.media.AudioAttributes to qualify your playback use case
08-30 12:07:44.253  3636  3636 W ServiceManagement: Waited one second for android.hardware.boot@1.0::IBootControl/default. Waiting another...
08-30 12:07:46.256  3636  3636 I chatty  : uid=0(root) bootctl identical 2 lines
08-30 12:07:47.259  3636  3636 W ServiceManagement: Waited one second for android.hardware.boot@1.0::IBootControl/default. Waiting another...
08-30 12:07:47.355  1235  1454 E LightsService: Light requested not available on this device. 2
08-30 12:07:48.259  3636  3636 W ServiceManagement: Waited one second for android.hardware.boot@1.0::IBootControl/default. Waiting another...
08-30 12:07:55.274  3636  3636 I chatty  : uid=0(root) bootctl identical 7 lines
08-30 12:07:56.275  3636  3636 W ServiceManagement: Waited one second for android.hardware.boot@1.0::IBootControl/default. Waiting another...
08-30 12:07:56.612  1235  1720 E LightsService: Light requested not available on this device. 2
08-30 12:07:56.696   971  3050 D audio_hw_primary: start_output_stream: enter: stream(0xf251c000)usecase(1: low-latency-playback) devices(0x2)
08-30 12:07:56.698   971  3050 D audio_hw_primary: select_devices for use case (low-latency-playback)
08-30 12:07:56.698   971  3050 D audio_hw_primary: select_devices: out_snd_device(2: speaker) in_snd_device(0: )
08-30 12:07:56.698   971  3050 I msm8916_platform: platform_check_and_set_codec_backend_cfg:becf: afe: bitwidth 16, samplerate 48000 channels 2, backend_idx 0 usecase = 1 device (speaker)
08-30 12:07:56.699   971  3050 D msm8916_platform: platform_split_snd_device: snd_device(2) num devices(1) new_snd_devices(0)
08-30 12:07:56.699   971  3050 I msm8916_platform: platform_check_and_set_codec_backend_cfg: becf: new_snd_devices[0] is speaker
08-30 12:07:56.699   971  3050 I msm8916_platform: platform_check_codec_backend_cfg:becf: afe: Codec selected backend: 0 current bit width: 16 sample rate: 48000 channels: 2 usecase 1 device (speaker)
08-30 12:07:56.699   971  3050 D msm8916_platform: platform_check_codec_backend_cfg:becf: afe: playback on codec device not supporting native playback set default Sample Rate(48k)
08-30 12:07:56.699   971  3050 I msm8916_platform: platform_check_codec_backend_cfg:becf: afe: Codec selected backend: 0 updated bit width: 16 sample rate: 48000 channels: 2
08-30 12:07:56.699   971  3050 D audio_hw_primary: check_usecases_codec_backend:becf: force routing 0
08-30 12:07:56.699   971  3050 E msm8916_platform: platform_check_backends_match: Invalid snd_device = 
08-30 12:07:56.699   971  3050 D audio_hw_primary: check_usecases_codec_backend:becf: (59) check_usecases curr device: speaker, usecase device: backends match 0
08-30 12:07:56.699   971  3050 D audio_hw_primary: check_usecases_codec_backend:becf: check_usecases num.of Usecases to switch 0
08-30 12:07:56.699   971  3050 D hardware_info: hw_info_append_hw_type : device_name = speaker
08-30 12:07:56.699   971  3050 D msm8916_platform: platform_split_snd_device: snd_device(2) num devices(0) new_snd_devices(0)
08-30 12:07:56.699   971  3050 D audio_hw_primary: enable_snd_device: snd_device(2: speaker)
08-30 12:07:56.699   971  3050 I soundtrigger: audio_extn_sound_trigger_update_device_status: device 0x2 of type 0 for Event 1, with Raise=0
08-30 12:07:56.699   971  3050 D audio_route: Apply path: speaker
08-30 12:07:56.699   971  3050 D audio_hw_primary: audio_is_true_native_stream_active:napb: (0) (low-latency-playback)id (1) sr 48000 bw (16) device speaker
08-30 12:07:56.699   971  3050 D soundtrigger: audio_extn_sound_trigger_update_stream_status: uc_id 1 of type 0 for Event 3, with Raise=0
08-30 12:07:56.699   971  3050 D audio_hw_utils: audio_extn_utils_send_app_type_cfg: usecase->out_snd_device speaker
08-30 12:07:56.699   971  3050 D msm8916_platform: platform_split_snd_device: snd_device(2) num devices(0) new_snd_devices(0)
08-30 12:07:56.700   971  3050 D audio_hw_utils: audio_extn_btsco_get_sample_rate:Not a BT SCO device, need not update sampling rate
08-30 12:07:56.700   971  3050 I audio_hw_utils: send_app_type_cfg_for_device PLAYBACK app_type 69937, acdb_dev_id 15, sample_rate 48000, snd_device_be_idx 43
08-30 12:07:56.700   995  1043 E ANDR-PERF-MPCTL: Invalid profile no. 0, total profiles 0 only
08-30 12:07:56.701   971  3050 D msm8916_platform: platform_split_snd_device: snd_device(2) num devices(1) new_snd_devices(0)
08-30 12:07:56.701   971  3050 D ACDB-LOADER: ACDB -> send_audio_cal, acdb_id = 15, path = 0, app id = 0x11131, sample rate = 48000
08-30 12:07:56.701   971  3050 D ACDB-LOADER: ACDB -> send_asm_topology
08-30 12:07:56.701   971  3050 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_AUDPROC_STREAM_TOPOLOGY_ID
08-30 12:07:56.701   971  3050 D ACDB-LOADER: ACDB -> send_adm_topology
08-30 12:07:56.701   971  3050 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_AUDPROC_COMMON_TOPOLOGY_ID
08-30 12:07:56.701   971  3050 D ACDB-LOADER: ACDB -> send_audtable
08-30 12:07:56.701   971  3050 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_AUDPROC_COMMON_TABLE_SIZE
08-30 12:07:56.701   971  3050 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_AUDPROC_COMMON_TABLE
08-30 12:07:56.701   971  3050 D ACDB-LOADER: ACDB -> AUDIO_SET_AUDPROC_CAL
08-30 12:07:56.701   971  3050 D ACDB-LOADER: ACDB -> send_audvoltable
08-30 12:07:56.701   971  3050 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_AUDPROC_VOL_STEP_TABLE_SIZE
08-30 12:07:56.701   971  3050 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_AUDPROC_GAIN_DEP_STEP_TABLE, vol index 5
08-30 12:07:56.701   971  3050 D ACDB-LOADER: ACDB -> AUDIO_SET_VOL_CAL cal type = 12
08-30 12:07:56.701   971  3050 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_AUDPROC_STREAM_TABLE_SIZE
08-30 12:07:56.702   971  3050 D ACDB-LOADER: ACDB -> send_audstrmtable
08-30 12:07:56.702   971  3050 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_AUDPROC_STREAM_TABLE_V2
08-30 12:07:56.702   971  3050 D ACDB-LOADER: ACDB -> audstrm_cal->cal_type.cal_data.cal_size = 16
08-30 12:07:56.702   971  3050 D ACDB-LOADER: ACDB -> send_afe_topology
08-30 12:07:56.702   971  3050 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_AFE_TOPOLOGY_ID
08-30 12:07:56.702   971  3050 D ACDB-LOADER: ACDB -> GET_AFE_TOPOLOGY_ID for adcd_id 15, Topology Id 1025e
08-30 12:07:56.702   971  3050 D ACDB-LOADER: ACDB -> send_afe_cal
08-30 12:07:56.702   971  3050 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_AFE_COMMON_TABLE_SIZE
08-30 12:07:56.702   971  3050 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_AFE_COMMON_TABLE
08-30 12:07:56.702   971  3050 D ACDB-LOADER: ACDB -> AUDIO_SET_AFE_CAL
08-30 12:07:56.702   971  3050 D ACDB-LOADER: ACDB -> send_hw_delay : acdb_id = 15 path = 0
08-30 12:07:56.702   971  3050 D ACDB-LOADER: ACDB -> ACDB_AVSYNC_INFO: ACDB_CMD_GET_DEVICE_PROPERTY
08-30 12:07:56.702   971  3050 D audio_hw_primary: enable_audio_route: apply mixer and update path: low-latency-playback speaker
08-30 12:07:56.702   971  3050 D audio_route: Apply path: low-latency-playback speaker
08-30 12:07:56.706   971  3050 D audio_hw_primary: select_devices: done
08-30 12:07:56.738  1864  2116 I vol.Events: writeEvent mute_changed STREAM_SYSTEM true
08-30 12:07:56.740  1864  2116 I vol.Events: writeEvent external_ringer_mode_changed silent
--------- beginning of system
08-30 12:07:56.744  1235  1235 V SettingsProvider: Notifying for 0: content://settings/global/zen_mode_config_etag
08-30 12:07:56.745  1235  1235 V SettingsProvider: Notifying for 0: content://settings/global/zen_mode
08-30 12:07:56.749  1864  2116 I vol.Events: writeEvent zen_mode_changed important_interruptions
08-30 12:07:56.757   971  3050 D msm8916_platform: platform_set_channel_map mixer_ctl_name:Playback Channel Map13
08-30 12:07:56.758   971  3050 D msm8916_platform: platform_set_channel_map: set mapping(1 2 0 0 0 0 0 0) for channel:2
08-30 12:07:56.758   971  3050 D audio_hw_primary: start_output_stream: exit
08-30 12:07:57.275  3636  3636 W ServiceManagement: Waited one second for android.hardware.boot@1.0::IBootControl/default. Waiting another...
08-30 12:07:58.276  3636  3636 W ServiceManagement: Waited one second for android.hardware.boot@1.0::IBootControl/default. Waiting another...
08-30 12:07:58.544  1864  1864 D StatusBar: disable<e i a s b h r c s > disable2<q i n >
08-30 12:07:58.621  1864  1864 D StatusBar: disable<e i a s b h r c s > disable2<q i n >
08-30 12:07:59.276  3636  3636 W ServiceManagement: Waited one second for android.hardware.boot@1.0::IBootControl/default. Waiting another...
08-30 12:07:59.360  1235  1789 I ActivityManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.android.camera2/com.android.camera.CameraLauncher (has extras)} from uid 10083
08-30 12:07:59.381  1235  1789 D PHH     : Overrode aspect ratio because pre-o to 1.86
08-30 12:07:59.423  1235  1789 D PHH     : Overrode aspect ratio because pre-o to 1.86
08-30 12:07:59.464  1014  1014 D SurfaceFlinger: duplicate layer name: changing Letterbox - bottom to Letterbox - bottom#1
08-30 12:07:59.490  1235  1427 I ActivityManager: Start proc 13672:com.android.camera2/u0a62 for activity com.android.camera2/com.android.camera.CameraLauncher
08-30 12:07:59.510 13672 13672 E android.camera: Not starting debugger since process cannot load the jdwp agent.
08-30 12:07:59.540  1235  1789 D PHH     : Overrode aspect ratio because pre-o to 1.86
08-30 12:07:59.552  1235  1789 D PHH     : Overrode aspect ratio because pre-o to 1.86
08-30 12:07:59.576 13672 13672 I android.camera: The ClassLoaderContext is a special shared library.
08-30 12:07:59.578 13672 13672 I android.camera: The ClassLoaderContext is a special shared library.
08-30 12:07:59.643 13672 13672 W SoundPool: Use of stream types is deprecated for operations other than volume control
08-30 12:07:59.643 13672 13672 W SoundPool: See the documentation of SoundPool() for what to use instead with android.media.AudioAttributes to qualify your playback use case
08-30 12:07:59.644 13672 13672 I CAM_OneCamFtrCnfgCrtr: CaptureModule? true
08-30 12:07:59.661  4162  4203 W Notification: Use of stream types is deprecated for operations other than volume control
08-30 12:07:59.661  4162  4203 W Notification: See the documentation of setSound() for what to use instead with android.media.AudioAttributes to qualify your playback use case
08-30 12:07:59.661 13672 13672 E SoundPool: error loading /product/media/audio/ui/camera_click.ogg
08-30 12:07:59.662 13672 13672 I CameraManagerGlobal: Connecting to camera service
08-30 12:07:59.667   973  2500 I QCamera : <HAL><INFO> getCameraInfo: 397: Camera id 0 API version 256
08-30 12:07:59.667   973  2500 I QCamera : <HAL><INFO> getCamInfo: 10404: camera 0 resource cost is 100
08-30 12:07:59.669   973  2500 I QCamera : <HAL><INFO> getCameraInfo: 397: Camera id 1 API version 256
08-30 12:07:59.669   973  2500 I QCamera : <HAL><INFO> getCamInfo: 10404: camera 1 resource cost is 100
08-30 12:07:59.672   973  2500 I QCamera : <HAL><INFO> getCameraInfo: 397: Camera id 0 API version 256
08-30 12:07:59.672   973  2500 I QCamera : <HAL><INFO> getCamInfo: 10404: camera 0 resource cost is 100
08-30 12:07:59.673   973  2500 I QCamera : <HAL><INFO> getCameraInfo: 397: Camera id 1 API version 256
08-30 12:07:59.673   973  2500 I QCamera : <HAL><INFO> getCamInfo: 10404: camera 1 resource cost is 100
08-30 12:07:59.690 13672 13698 I OMXClient: IOmx service obtained
08-30 12:07:59.691  1124  1933 I OMXMaster: makeComponentInstance(OMX.google.vorbis.decoder) in omx@1.0-service process
08-30 12:07:59.693 13672 13672 D OpenGLRenderer: Skia GL Pipeline
08-30 12:07:59.696  1124  5151 E OMXNodeInstance: getConfig(0xe7819420:google.vorbis.decoder, ConfigAndroidVendorExtension(0x6f100004)) ERROR: Undefined(0x80001001)
08-30 12:07:59.700   969   969 W AshmemAllocator: ashmem_create_region(32768) returning hidl_memory(0x790622b530, 32768)
08-30 12:07:59.713   969   969 I chatty  : uid=1000(system) allocator@1.0-s identical 6 lines
08-30 12:07:59.713   969   969 W AshmemAllocator: ashmem_create_region(32768) returning hidl_memory(0x790622b530, 32768)
08-30 12:07:59.716 13672 13672 W CAM_Log : Tag RoundedThumbnailView is 1 chars longer than limit.
08-30 12:07:59.785 13672 13672 I CAM_Profiler: [164.360ms][ui] GUARD: CameraActivity.onCreateTasks - [103.517ms] setContentView()
08-30 12:07:59.800 13672 13690 W CAM2PORT_AndCamAgntImp: Releasing camera without any camera opened.
08-30 12:07:59.814   973  2500 I QCamera : <HAL><INFO> getCameraInfo: 397: Camera id 0 API version 256
08-30 12:07:59.814   973  2500 I QCamera : <HAL><INFO> getCamInfo: 10404: camera 0 resource cost is 100
08-30 12:07:59.817 13672 13672 D AndroidRuntime: Shutting down VM
--------- beginning of crash
08-30 12:07:59.818 13672 13672 E AndroidRuntime: FATAL EXCEPTION: main
08-30 12:07:59.818 13672 13672 E AndroidRuntime: Process: com.android.camera2, PID: 13672
08-30 12:07:59.818 13672 13672 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.android.camera2/com.android.camera.CameraActivity}: java.lang.RuntimeException: Camera is being used after Camera.release() was called
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2913)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3048)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1808)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:193)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6669)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: Caused by: java.lang.RuntimeException: Camera is being used after Camera.release() was called
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.hardware.Camera._enableShutterSound(Native Method)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.hardware.Camera.updateAppOpsPlayAudio(Camera.java:1770)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.hardware.Camera.initAppOps(Camera.java:582)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.hardware.Camera.<init>(Camera.java:575)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.hardware.Camera.getEmptyParameters(Camera.java:2130)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.hardware.camera2.legacy.LegacyMetadataMapper.createCharacteristics(LegacyMetadataMapper.java:151)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.hardware.camera2.CameraManager.getCameraCharacteristics(CameraManager.java:274)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at com.android.camera.one.v2.Camera2OneCameraManagerImpl.findFirstCameraIdFacing(Camera2OneCameraManagerImpl.java:159)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at com.android.camera.one.v2.Camera2OneCameraManagerImpl.findFirstBackCameraId(Camera2OneCameraManagerImpl.java:135)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at com.android.camera.one.v2.Camera2OneCameraManagerImpl.findCameraId(Camera2OneCameraManagerImpl.java:128)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at com.android.camera.one.v2.Camera2OneCameraManagerImpl.findFirstCameraFacing(Camera2OneCameraManagerImpl.java:102)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at com.android.camera.CaptureModule.updateCameraCharacteristics(CaptureModule.java:390)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at com.android.camera.CaptureModule.init(CaptureModule.java:417)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at com.android.camera.CameraActivity.onCreateTasks(CameraActivity.java:1627)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at com.android.camera.util.QuickActivity.onCreate(QuickActivity.java:114)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:7136)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:7127)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2893)
08-30 12:07:59.818 13672 13672 E AndroidRuntime: 	... 11 more
08-30 12:07:59.823  1235  1966 W ActivityManager:   Force finishing activity com.android.camera2/com.android.camera.CameraLauncher
08-30 12:07:59.839  1235  1421 I ActivityManager: Showing crash dialog for package com.android.camera2 u0
08-30 12:07:59.860  1235  2952 I OpenGLRenderer: Initialized EGL, version 1.4
08-30 12:07:59.862  1235  2952 D OpenGLRenderer: Swap behavior 2
08-30 12:07:59.906   971  1790 D audio_hw_primary: out_standby: enter: stream (0xf251c000) usecase(1: low-latency-playback)
08-30 12:07:59.951   971  1790 D audio_hw_primary: disable_audio_route: reset and update mixer path: low-latency-playback speaker
08-30 12:07:59.952   971  1790 D soundtrigger: audio_extn_sound_trigger_update_stream_status: uc_id 1 of type 0 for Event 2, with Raise=0
08-30 12:07:59.952   971  1790 D hardware_info: hw_info_append_hw_type : device_name = speaker
08-30 12:07:59.952   971  1790 D audio_hw_primary: disable_snd_device: snd_device(2: speaker)
08-30 12:07:59.952   971  1790 D msm8916_platform: platform_split_snd_device: snd_device(2) num devices(0) new_snd_devices(0)
08-30 12:07:59.952   971  1790 I soundtrigger: audio_extn_sound_trigger_update_device_status: device 0x2 of type 0 for Event 0, with Raise=0
08-30 12:07:59.952   971  1790 D audio_hw_primary: out_standby: exit
08-30 12:08:00.277  3636  3636 W ServiceManagement: Waited one second for android.hardware.boot@1.0::IBootControl/default. Waiting another...
08-30 12:08:00.332  1235  1418 W ActivityManager: Activity pause timeout for ActivityRecord{8335f6d u0 com.android.camera2/com.android.camera.CameraLauncher t58 f}
08-30 12:08:00.509  1864  1883 I ndroid.systemu: Background concurrent copying GC freed 160191(6MB) AllocSpace objects, 0(0B) LOS objects, 49% free, 7MB/15MB, paused 1.164ms total 107.125ms
08-30 12:08:01.278  3636  3636 W ServiceManagement: Waited one second for android.hardware.boot@1.0::IBootControl/default. Waiting another...
08-30 12:08:01.539  1235  1966 W ActivityManager:   Force finishing activity com.android.camera2/com.android.camera.CameraLauncher
08-30 12:08:01.546  1235  2952 D OpenGLRenderer: endAllActiveAnimators on 0x7bcd595100 (RippleDrawable) with handle 0x7bdc10d580
08-30 12:08:01.549  1014  1465 W SurfaceFlinger: Attempting to destroy on removed layer: AppWindowToken{54fcc33 token=Token{4107a2 ActivityRecord{8335f6d u0 com.android.camera2/com.android.camera.CameraLauncher t58}}}#0
08-30 12:08:01.591  1235  1966 I ActivityManager: Killing 13672:com.android.camera2/u0a62 (adj 900): crash
08-30 12:08:01.600  1235  1428 W libprocessgroup: kill(-13672, 9) failed: No such process
08-30 12:08:01.637  1235  1428 W libprocessgroup: kill(-13672, 9) failed: No such process
08-30 12:08:01.642  1235  1428 W libprocessgroup: kill(13672, 9) failed: No such process
08-30 12:08:01.642   826   826 I Zygote  : Process 13672 exited due to signal (9)
08-30 12:08:01.653  1235  1428 W libprocessgroup: kill(-13672, 9) failed: No such process
08-30 12:08:01.653  1235  1428 I libprocessgroup: Successfully killed process cgroup uid 10062 pid 13672 in 53ms
08-30 12:08:01.758  1014  1056 W SurfaceFlinger: Attempting to set client state on removed layer: Dim Layer for - mAboveAppWindowsContainers#0
08-30 12:08:01.758  1014  1056 W SurfaceFlinger: Attempting to destroy on removed layer: Dim Layer for - mAboveAppWindowsContainers#0
08-30 12:08:01.780  1014  1056 W SurfaceFlinger: Attempting to destroy on removed layer: 5ec8b0 Application Error: com.android.camera2#0
08-30 12:08:02.279  3636  3636 W ServiceManagement: Waited one second for android.hardware.boot@1.0::IBootControl/default. Waiting another...
```

--- a/rw-system.sh
+++ b/rw-system.sh
@@ -293,7 +293,7 @@ if getprop ro.vendor.build.fingerprint | grep -qiE '^samsung'; then
     fi
 fi
 
-if getprop ro.vendor.build.fingerprint | grep -qE '^xiaomi/daisy/daisy_sprout:8.1.0/OPM.*'; then
+if getprop ro.vendor.build.fingerprint | grep -qE '^xiaomi/(daisy|wayne)/(daisy|wayne).*'; then
     # Fix camera on DND, ugly workaround but meh
     setprop audio.camerasound.force true
 fi

